browser/src/core/Socket.js

```diff
@@ -481,7 +481,7 @@ app.definitions.Socket = L.Class.extend({
 		imgBytes = e.imgBytes;
 
 		this._logSocket('INCOMING', textMsg);
-
+		
```

Comment:
This was probably not intentionally, could you please remove `Socket.js` from this PR